src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -28,18 +28,17 @@ public class AddCommand extends Command {
     /**
      * Usage message for the add command.
      */
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book.\n"
             + "Parameters: "
             + PREFIX_NAME + "NAME "
             + "[" + PREFIX_TAG + "TAG]..."
-            + "(optional: [" + PREFIX_CATEGORY + "CATEGORY NAME] "
-            + "[" + PREFIX_DESCRIPTION + "DESCRIPTION])\n"
```

Comment:
This is due to the fact that users might type (optional:) in their input. This will cause an unintended outcome of the error message and thus considered a bug.